### ISSUE TITLE
Default test template for selector is failed automatically.

### ISSUE THREAD

**Issue by chaintan17** [mxstbr/react-boilerplate#732]
When create new component including test template.
Without changing anything, the test suite become failed as below

```
START:
17 07 2016 20:52:22.145:INFO [karma]: Karma v1.1.1 server started at http://localhost:9876/
17 07 2016 20:52:22.148:INFO [launcher]: Launching browser Chrome with unlimited concurrency
17 07 2016 20:52:22.165:INFO [launcher]: Starting browser Chrome
17 07 2016 20:52:27.468:INFO [Chrome 51.0.2704 (Mac OS X 10.10.5)]: Connected on socket /#JJNDJPt-Esy8BKyyAAAA with id 64471088
Chrome 51.0.2704 (Mac OS X 10.10.5) ERROR
  Uncaught TypeError: (0 , _selectors.selectCheckout) is not a function
  at webpack:///app/containers/Checkout/tests/selectors.test.js:9:632 <- test-bundler.js:63465

Finished in 2.117 secs / 0 secs

SUMMARY:
✔ 0 tests completed
npm ERR! Test failed.  See above for more details.
```

Is it should be fixed? If yes, I'm trying to fix it.... :) `#grabs-for-grabs` `#first-timers-only`

**Comment by gihrig** [comment]
All generated tests fail as initially created. They are basic stubs, sort of reminders that you need to write tests for you your new container.

It would be nice if all generated tests were more complete. For example, present a failing test not an error.

**Comment by mxstbr** [comment]
Fixed in #733 by the amazing @chaintan17, thanks so much!
